experimental/ag-ui/front-end/src/index.css

```
@@ -1,15 +1,190 @@
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border: 0;
+}
+
+/* ── Utility Animations ── */
+@keyframes fadeIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)